lms/services/jstor/_article_metadata.py

```diff
@@ -64,28 +64,31 @@ def content_status(self) -> str:
         return "no_access"
 
     @property
-    def title(self) -> str:
+    def titles(self) -> dict:
         # Reviews of other works may not have a title of their own, but we can
         # generate one from the reviewed work's metadata.
         if reviewed_works := self._data.get("reviewed_works"):
-            title = f"Review: {reviewed_works[0]}"
+            return {"title": self._strip_html_tags(f"Review: {reviewed_works[0]}")}
+
+        if titles := self._get_titles("title", "subtitle"):
+            return titles
+
+        return {"title": "[Unknown title]"}
+
+    def _get_titles(self, title_key, subtitle_key):
+        titles = {}
 
         # Journal articles, book chapters and research reports have a title
         # field with a single entry.
-        elif title := self._data.get("title"):
+        if title := self._data.get(title_key):
+            titles["title"] = self._strip_html_tags(title)
 
             # Some articles have a subtitle which needs to be appended for the
             # title to make sense.
-            if subtitle := self._data.get("subtitle"):
-                # Some titles include a trailing ':' delimiter, some do not.
-                title = f"{title.rstrip(':')}: {subtitle}"
```

Comment:
How much grief would it be to add back the colon-stripping off of the end of `title` here? That's messy data and my light preference would be to have the backend take care of it, but it _can_ be done on the frontend if needed.

Reply:
Not a problem

Reply:
We should now strip this again.